deploy/helm/scf/templates/ops.yaml

```diff
@@ -14,7 +14,7 @@ metadata:
     helm.sh/chart: {{ include "scf.chart" .Root }}
 data:
   ops: |-
-    {{- .Root.Files.Get .Path | nindent 4 -}}
+    {{- tpl (.Root.Files.Get .Path) .Root | nindent 4 -}}
```

Comment:
+1

With this, I think we can stop creating lots of implicit vars.

Reply:
Yes, very useful and seamless